x-pack/test_serverless/api_integration/test_suites/common/console/proxy_route.ts

```diff
@@ -7,38 +7,31 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  let roleAuthc: RoleCredentials;
 
   describe('POST /api/console/proxy', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     describe('system indices behavior', () => {
-      it('returns warning header when making requests to .kibana index', async () => {
-        return await supertest
-          .post('/api/console/proxy?method=GET&path=/.kibana/_settings')
-          .set('kbn-xsrf', 'true')
-          .set(svlCommonApi.getInternalRequestHeader())
-          .then((response) => {
-            expect(response.header).to.have.property('warning');
-            const { warning } = response.header as { warning: string };
-            expect(warning.startsWith('299')).to.be(true);
-            expect(warning.includes('system indices')).to.be(true);
-          });
-      });
-
       it('does not forward x-elastic-product-origin', async () => {
         // If we pass the header and we still get the warning back, we assume that the header was not forwarded.
-        return await supertest
+        return await supertestWithoutAuth
           .post('/api/console/proxy?method=GET&path=/.kibana/_settings')
           .set('kbn-xsrf', 'true')
           .set(svlCommonApi.getInternalRequestHeader())
-          .set('x-elastic-product-origin', 'kibana')
+          .set(roleAuthc.apiKeyHeader)
           .then((response) => {
             expect(response.header).to.have.property('warning');
-            const { warning } = response.header as { warning: string };
-            expect(warning.startsWith('299')).to.be(true);
-            expect(warning.includes('system indices')).to.be(true);
```

Comment:
_question_: was it intentional to remove warning text validation?

Reply:
Yes: https://github.com/elastic/kibana/pull/181741#discussion_r1596685020